chainer/variable.py

```diff
@@ -529,3 +529,22 @@ def __hash__(self):
         return super(Variable, self).__hash__()
 
     __array_priority__ = 200
+
+
+def get_array(arg):
+    """Returns ndarray data.
+
+    This method is helper function for :class:`~chainer.Variable`.
```

Comment:
Write a description about this method like
"It returns an ndarray which a given variable holds."